GettingStarted.md

```diff
@@ -0,0 +1,103 @@
+# SailfishOS Chum - Getting Started with OBS
+
+This document summarises the proces of building your first package on 
+OBS.  It only skims the surface of OBS and is not a manual.
+
+To start building your packages on OBS you will require an account.
+Follow the process in the README and contact 'lbt' for an account to 
+be set up.
+
+When your account is set up, you will have a "Home Project".  This is where
+you can start creating packages.  The URL for your home project will be:
+https://build.merproject.org/project/show/home:<username>
+
+## Creating a package on OBS
+
+Projects contain packages which are built against target repositories
+(sailfish versions).  To create your first package, perform the following
+steps:
+
+1. Click "Create Package"
+2. On the new package page, give the package a name and click "Save Changes"
+   * eg harbour-mypackage<br>
+You now need to add files to the package to allow OBS to get the source and
+build it.
+3. Create a file on your computer called _service with th e following content:
+        <services>
+          <service name="tar_git">
+            <param name="url">URL TO PROJECT</param>
+            <param name="branch">master</param>
+            <param name="revision"></param>
+            <param name="token"/>
+            <param name="debian">N</param>
+            <param name="dumb">N</param>
+          </service>
+        </services>`
```

Comment:
````suggestion
```
        <services>
          <service name="tar_git">
            <param name="url">URL TO PROJECT</param>
            <param name="branch">master</param>
            <param name="revision"></param>
            <param name="token"/>
            <param name="debian">N</param>
            <param name="dumb">N</param>
          </service>
        </services>`
```
````
These lines need surrounding in "code block" formatters. Perhaps it would also help to clarify what `N` is.

Reply:
Probably those last 3 params are not even needed